test/views/TaskListViewSpec.scala

```diff
@@ -35,6 +36,24 @@ class TaskListViewSpec extends ViewBehaviours with TaskListViewBehaviours {
   private def sections(answers: UserAnswers) =
     new RegistrationProgress(new TaskListNavigator()).sections(answers)
 
+  private def trustDetailsSectionStatus(answers: UserAnswers) =
+    new RegistrationProgress(new TaskListNavigator()).trustDetailsSectionStatus(answers)
```

Comment:
@jassalrichy  create the object one time outside of method as it is the same in each method.

Reply:
Amended.